common/redis/templates/deployment.yaml

```diff
@@ -32,6 +32,20 @@ spec:
         checksum/secrets: {{ include (print .Template.BasePath "/secrets.yaml") . | sha256sum }}
         {{- end }}
     spec:
+      {{- if not .Values.redisPassword }}
+      initContainers:
+      - name: generate-secrets
+        image: {{ $registry }}/shared-app-images/alpine-kubectl:3.21-latest
```

Comment:
I would use latest-latest here

Reply:
This is copied from postgresql-ng. Should we switch to latest-latest over there as well?